itou/www/apply/forms.py

```diff
@@ -349,45 +349,65 @@ class SiaePrescriberFilterJobApplicationsForm(FilterJobApplicationsForm):
     Job applications filters common to SIAE and Prescribers.
     """
 
-    sender = forms.MultipleChoiceField(
-        required=False, label=_("Personne"), widget=Select2MultipleWidget
+    senders = forms.MultipleChoiceField(
+        required=False, label=_("Nom"), widget=Select2MultipleWidget
+    )
+
+    job_seekers = forms.MultipleChoiceField(
+        required=False, label=_("Candidat"), widget=Select2MultipleWidget
     )
 
     def __init__(self, job_applications_qs, *args, **kwargs):
         self.job_applications_qs = job_applications_qs
         super().__init__(*args, **kwargs)
-        self.fields["sender"].choices += self.get_sender_choices()
+        self.fields["senders"].choices += self._get_choices_for("sender")
+        self.fields["job_seekers"].choices = self._get_choices_for("job_seeker")
+
+    def _get_choices_for(self, user_type):
+        users = self.job_applications_qs.get_unique_fk_objects(user_type)
+        users = [user for user in users if user.get_full_name()]
+        users = [(user.id, user.get_full_name().title()) for user in users]
+        return sorted(users, key=lambda l: l[1])
+
+    def _humanize_multiple_choice_for_users(self, user_ids, field_name):
+        values = [
+            get_user_model().objects.get(id=int(user_id)).get_full_name().title()
+            for user_id in user_ids
+        ]
```

Comment:
J'ai l'impression que ça fait une requête SQL par user non? Si c'est le cas, pas top. Vérifiable dans ton local dev ici:
![image](https://user-images.githubusercontent.com/10533583/75251745-ee929d00-57e3-11ea-960d-32e761422a28.png)

Tu peux faire une seule requête avec un truc du genre

`[user.get_full_name().title() for user in get_user_model().objects.filter(pk__in([int(user_id) for user_id in user_ids]))]`

Reply:
Olala, oui ! Merci ! Je change ça.